src/Middleware/Diagnostics/src/ExceptionHandler/ExceptionHandlerServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Builder;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods for the exception handler middleware.
+    /// </summary>
+    public static class ExceptionHandlerServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Adds services and options for the exception handler middleware.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection"/> for adding services.</param>
+        /// <param name="configureOptions">A delegate to configure the <see cref="ExceptionHandlerOptions"/>.</param>
+        /// <returns></returns>
+        public static IServiceCollection AddExceptionHandler(this IServiceCollection services, Action<ExceptionHandlerOptions> configureOptions)
```

Comment:
Update this message to reference this extension
https://github.com/dotnet/aspnetcore/blob/36856ca8f9932b88757bbf24904bf36868b4d727/src/Middleware/Diagnostics/src/Resources.resx#L251